app/modules/progress/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+"""
+Progress module
```

Comment:
Would be helpful if there was a description of exactly what this is Progress of?
The asset group sighting? The detection task? The Id task? The full ia pipeline? 
At the moment it's ambiguous and could cause confusion.

Reply:
One thing, when you do refactor the completion of the overall status, remember to remove this version.
https://github.com/WildMeOrg/houston/blob/main/app/modules/asset_groups/models.py#L322 
https://github.com/WildMeOrg/houston/blob/main/app/modules/sightings/models.py#L1200
and all of their usages

Reply:
Noted and removed